ci/unit/test_main_node.sh

```diff
@@ -30,8 +30,9 @@ fi
 sudo chown root /usr/bin/daos_admin
 sudo chmod 4755 /usr/bin/daos_admin
 /bin/rm "$DAOS_BASE/install/bin/daos_admin"
-sudo ln -sf "$SL_PREFIX/share/spdk/scripts/setup.sh" /usr/share/spdk/scripts
-sudo ln -sf "$SL_PREFIX/share/spdk/scripts/common.sh" /usr/share/spdk/scripts
+sudo mkdir -p /usr/share/spdk/scripts/
+sudo ln -sf "$SL_PREFIX/share/spdk/scripts/setup.sh" /usr/share/spdk/scripts/
+sudo ln -sf "$SL_PREFIX/share/spdk/scripts/common.sh" /usr/share/spdk/scripts/
```

Comment:
What's changed here other than the extra mkdir call?

Reply:
Just trailing `/` to make it explicit that the link expects the destination to be a pre-existing directory and not silently creating a `scripts` file.  Belt and suspenders.